tekton/pipeline_run_test.go

```diff
@@ -262,4 +269,58 @@ var _ = Describe("PipelineRun", func() {
 			})
 		})
 	})
+
+	Context("When calling getPipelineRef", func() {
+		It("should return a PipelineRef without resolver if the releaseStrategy contains no bundle", func() {
+			releaseStrategy := &v1alpha1.ReleaseStrategy{
+				Spec: v1alpha1.ReleaseStrategySpec{
+					Pipeline:              "release-pipeline",
+					Bundle:                "",
+					Policy:                "testpolicy",
+					PersistentVolumeClaim: persistentVolumeClaim,
```

Comment:
I don't think this is needed as it is an optional field and doesn't matter in this function